plugins/generator-1.15.2/forge-1.15.2/templates/json/txblock/end_rod.json.ftl

```diff
@@ -0,0 +1,49 @@
+<#-- @formatter:off -->
+{
+    "parent": "block/block",
+    "display": {
+        "head": {
+            "rotation": [ -60, 0, 0 ],
+            "translation": [ 0, 5, -9],
+            "scale":[ 1, 1, 1]
+        },
+        "thirdperson_righthand": {
+            "rotation": [ 0, 0, 0 ],
+            "translation": [ 0, 0, 0],
+            "scale": [ 0.375, 0.375, 0.375 ]
+        }
+    },
+    "ambientocclusion": false,
+    "textures": {
+        "end_rod": "${modid}:blocks${data.textureFront?has_content?then(data.textureFront, data.texture)}",
```

Comment:
End rod uses data.texture so this check is not needed as it always defaults to data.texture. Fix this for both generators.